sys/dts/litex-riscv.dts

```diff
@@ -0,0 +1,124 @@
+
+/dts-v1/;
+
+/ {
+        #address-cells = <1>;
+        #size-cells = <1>;
+
+
+        chosen {
+            linux,initrd-start = <0x42000000>;
+            linux,initrd-end = <0x45000000>;
```

Comment:
This should depend on the actual size of ramdisk attached by Renode.

Reply:
Although you're right that the size of the ram disk should be determined automatically, obtaining the value from RENODE's platform description may not be the best idea. We agreed that the size should be determined at launch time using device tree overlays (similar to #1258).